src/templatizer/dom-if.html

```diff
@@ -30,63 +30,78 @@
    * Set the `restamp` property to true to force the stamped content to be
    * created / destroyed when the `if` condition changes.
    */
-  Polymer({
-
-    is: 'dom-if',
+  class DomIf extends Polymer.Element {
+
+    static get is() { return 'dom-if' }
```

Comment:
You didn't use this when defining and there's no template. We should be able to omit it here then right?